api/src/main/java/com/exelaration/abstractmemery/services/implementations/MemeServiceImpl.java

```diff
@@ -26,4 +27,24 @@ public Meme save(Meme meme) {
       return null;
     }
   }
+
+  public ArrayList<String> getMemes() {
+    ArrayList<String> memeNames = memeMetadataService.getMemes();
+    if (memeNames == null) {
+      return null;
+    }
+    ArrayList<String> memeURLs = new ArrayList<String>();
+    String directory = "/app/src/main/resources/memes/";
+    for (String memeName : memeNames) {
+      try {
+        String meme = fileStorageService.getFileURL(memeName, directory);
```

Comment:
Passing in the directory isn't necessary here.  The `fileStorageService` should know where to find the images, we just need to tell it what image to give us.